client/my-sites/github-deployments/controller.tsx

```diff
@@ -89,14 +91,22 @@ export const redirectHomeIfIneligible: Callback = ( context, next ) => {
 	const siteId = getSelectedSiteId( state );
 	const siteSlug = getSelectedSiteSlug( state );
 	const site = getSelectedSite( state );
+	const hasSftpFeature = siteHasFeature( state, siteId, FEATURE_SFTP );
 	const isAtomicSite = !! site?.is_wpcom_atomic || !! site?.is_wpcom_staging_site;
 	const isJetpackNonAtomic = ! isAtomicSite && !! site?.jetpack;
 
 	if ( isEnabled( 'layout/dotcom-nav-redesign-v2' ) ) {
+		// Go to the Dev Tools screen for Simple sites transferable to Atomic.
+		if ( ! isAtomicSite && hasSftpFeature ) {
```

Comment:
What if we put this in a new middleware? Something like `redirectToDevToolsIfNotAtomic`, something like

https://github.com/Automattic/wp-calypso/blob/368cc08c379368a68f4ac7dd7c2e6839615c612b/client/controller/index.web.js#L234-L246

I imagine it will be useful e.g. for site monitoring.

Reply:
Sure! I've updated the PR description accordingly.